lib/playground.dart

```diff
@@ -149,15 +196,22 @@ class Playground {
     _context.onDartDirty.listen((_) => dartBusyLight.on());
     _context.onDartReconcile.listen((_) => _performAnalysis());
 
+    _finishedInit();
+  }
+
+  _finishedInit() {
+    // Clear the splash.
     DSplash splash = new DSplash(querySelector('div.splash'));
     splash.hide();
 
-    // TODO: This will need to be re-worked.
-    // Run the current contents.
-    Timer.run(() {
-      _handleRun();
-      _performAnalysis();
-    });
+    // TODO: I really want to be able to have paths without hashes in them.
+    // So, dartpad.foo/123456af rather then dartpad.foo/#123456af.
+    Router router = new Router(useFragment: true);
```

Comment:
@lukechurch, this is a workaround to having a true `dartpad.foo/123456af` path - we use a hash in the url, and everything after is the gist id.